umake/frameworks/go.py

```diff
@@ -42,13 +42,11 @@ def __init__(self):
 
 class GoLang(umake.frameworks.baseinstaller.BaseInstaller):
 
-    def __init__(self, category):
+    def __init__(self, **kwargs):
         super().__init__(name="Go Lang", description=_("Google compiler (default)"), is_category_default=True,
-                         category=category, only_on_archs=['i386', 'amd64'],
-                         download_page="https://golang.org/dl/",
-                         checksum_type=ChecksumType.sha1,
-                         dir_to_decompress_in_tarball="go",
-                         required_files_path=[os.path.join("bin", "go")])
+                         only_on_archs=['i386', 'amd64'], download_page="https://golang.org/dl/",
```

Comment:
Sorry for my formatting, I'm not yet accustomed to participate to a project like that. It's entirely my fault, I agree with you I just need to modify the lines of code that concern me! I will undo my modifications ;)